src/main/java/mate/academy/dto/category/CategoryDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dto.category;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record CategoryDto(
+        Long id,
+        @NotBlank
```

Comment:
is it response? don't need to validate it

Reply:
Sorry. You are right. This is just lazy student initialization.